### DISCUSSION TITLE
How can we test 5G in Magma?

### DISCUSSION THREAD

**Discussion by ShubhamTatvamasi** [magma/magma#7022] [General]
What are all the steps we can take for testing 5G from **magma-5G** branch?

```bash
git clone -b magma-5G https://github.com/magma/magma.git
```
Magma 5G - https://drive.google.com/drive/folders/18dOVLtSQwlgKUSclDXEsHYPpD1DhbcDO

**Comment by ShubhamTatvamasi** [comment]
@sureshgorijavolu

**Reply by sureshgorijavolu** [reply]
hi Shubham,
in a few days 5G SA Test team will draft a page on the step-by-step procedure to be followed to test 5G SA AGW.

**Reply by toddeye** [reply]
@sureshgorijavolu Is the step-by-step procedure available now?

**Reply by MAAHamid** [reply]
@sureshgorijavolu Is that planned document is ready? I have noticed the docs on the above sharepoint weren't updated since Jul 2021

**Reply by manishks99** [reply]
@sureshgorijavolu , Can you share URL of the document?

Did you meant following URL (https://docs.magmacore.org/docs/next/lte/s1ap_tests) document to test 5G SA AGW.
